src/main/java/org/folio/circulation/domain/StoredRequestRepresentation.java

```diff
@@ -70,8 +72,7 @@ private static JsonObject identifierToJson(Identifier identifier) {
 
   private static void addStoredRequesterProperties(JsonObject request, User requester) {
     if (requester == null) {
-      String msg = "Unable to add requester properties to the request: {}, requester is null.";
-      log.info(msg, request.getString("id"));
+      logUnableToAddNullPropertiesToTheRequest("requester", request.getString("id"));
```

Comment:
Can we pass the whole request and extract so that we don't have to do `request.getString("id")` every time?
```suggestion
      logUnableToAddNullPropertiesToTheRequest("requester", request);
```

Reply:
I've used it for request as json and request as Request type, but i think I can update to just use json everywhere

Reply:
updated